.github/workflows/npm-publish.yml

```
@@ -12,17 +10,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
-    - name: Publish if version has been updated
-      uses: pascalgn/npm-publish-action@1.3.6
-      with: # All of theses inputs are optional
-        create_tag: "false"
-        # We don't care what the commit message is since a new tag is the trigger
-        # Need to specify this to avoid the default
-        commit_pattern: ".*"
-        workspace: "."
-        publish_command: "yarn"
-        publish_args: "--non-interactive"
-      env: # More info about the environment variables in the README
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Leave this as is, it's automatically generated
-        NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }} # You need to set this in your repo settings
+    - uses: actions/checkout@v2
+    - name: Publish
+    - uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+          registry-url: 'https://registry.npmjs.org'
+          - run: npm install
+          - run: npm publish
+            env:
+              NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

I think the indentation is slightly off (eg. the `- run` steps at the bottom should be at the same level as the `setup-node` item as they're steps)
Also at the moment you have 2 steps with just a name (Checkout repository and Publish) which don't do anything, so need to either remove the name key or remove the second `-` so the `uses` and `name` are part of the same step
```suggestion
      - name: Checkout repository
        uses: actions/checkout@v2
      - name: Publish
        uses: actions/setup-node@v2
        with:
          node-version: 12.x"
          registry-url: "https://registry.npmjs.org"
      - run: npm install
      - run: npm publish
        env:
            NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

---

Have pushed a revised workflow.